src/IonDecimal.ts

```diff
@@ -17,7 +17,22 @@ import {_sign} from "./util";
 
 /**
  * This class provides the additional semantics necessary for decimal values.
- * Note that range of an exponent is limited to +/- 15 digits.
+ *
+ * Unlike IEEE-754 floating point representation, Decimal values can represent numeric values without loss of precision.
+ * They combine an arbitrarily large integer value (the "coefficient") with a second integer representing its scale
+ * (the "exponent") using the following formula:
+ *
+ *     value = coefficient * (10 ^ exponent)
+ *
+ * Here are some examples:
+ *
+ *      coefficient     exponent        value
+ *             0           0                0
+ *           314          -2                3.14
+ *           -76           1              -76
```

Comment:
something's off in this example:  `-76d1` would be `-760` (vs. `-76d0`, which would be `-76`)

Reply:
Whoops, good catch!